src/transformers/models/clipseg/configuration_clipseg.py

```diff
@@ -270,8 +272,7 @@ class CLIPSegConfig(PretrainedConfig):
             The dropout ratio for the attention probabilities.
         decoder_hidden_act (`str` or `function`, *optional*, defaults to `"quick_gelu"`):
             The non-linear activation function (function or string) in the encoder and pooler. If string, `"gelu"`,
-            `"relu"`, `"selu"` and `"gelu_new"` ``"quick_gelu"` are supported. layer_norm_eps (`float`, *optional*,
-            defaults to 1e-5): The epsilon used by the layer normalization layers.
```

Comment:
remove `layer_norm_eps` for the top level `CLIPSegConfig`, as it is never used, and to be aligned with other CLIP-like models